Cargo.toml

```diff
@@ -34,7 +34,3 @@ env_logger = "0.6"
 glsl-to-spirv = "0.1"
 log = "0.4"
 png = "0.15"
-
-[patch.crates-io]
```

Comment:
I added these because I think they are useful. Whenever you want to test a local gfx override, you can see how this is done, only edit a few characters and good to go.